### ISSUE TITLE
ImportError: Plotly express requires pandas to be installed.

### ISSUE THREAD

**Issue by sciunto** [emmanuelle/surface-tension#1]
Funny. In a clean environment;
```
Traceback (most recent call last):
  File "app.py", line 10, in <module>
    import plotly.express as px
  File "/home/fr/.local/share/virtualenvs/emma/lib/python3.8/site-packages/plotly/express/__init__.py", line 10, in <module>
    raise ImportError(
ImportError: Plotly express requires pandas to be installed.
```

**Comment by emmanuelle** [comment]
Closed by 90599cc .

**Comment by emmanuelle** [comment]
Pandas is an optional dependency only for plotly but it is required by plotly express, I always forget about it...
